### PR TITLE
added autopay and fail-safe behaviour to swarm documentation

### PR DESCRIPTION
**Pull request by mortonbits**
## Motivation

As discussed in our discord channel it would be helpful if the "fail-save" behaviour of validator selection is described in context of swarm and autopay.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan

This PR only covers documentation update. Please check the updated section at the end of the documentation if it is o.k.

### CODE REVIEW

File: ol/documentation/devs/swarm_qa_tools.md [hunk 1]
```diff
@@ -187,3 +187,23 @@ cargo r -p txs -- --swarm-path=$HOME/swarm_temp/ --swarm-persona=bob --save-path
 cd $HOME/libra
 cargo r -p txs -- --swarm-path=$HOME/swarm_temp/ --swarm-persona=bob relay --relay-file ./noop_tx.json
 ```
+
+### swarm and autopay transactions
+
+When you experiment with swarm and the autopay feature, be aware that there is a "fail-save" build into the validator set election. If we don't get 4 validators who qualify to be in the validator set, then the system does not change the validator set. So often in swarm for dev purposes we will have 4 or fewer nodes, so we don't get an accurate picture of the reconfiguration.
```
Comment (lpgeiger) [inline ol/documentation/devs/swarm_qa_tools.md:193]: spelling :) "fail-safe built"

File: ol/documentation/devs/swarm_qa_tools.md [hunk 2]
```diff
@@ -187,3 +187,23 @@ cargo r -p txs -- --swarm-path=$HOME/swarm_temp/ --swarm-persona=bob --save-path
...
+
+In the "real world", a validator node which has no autopay set up will be thrown out of the validator set, but in swarm, as no node has autopay set up per default, no node will be thrown out.
```
Comment (lpgeiger) [inline ol/documentation/devs/swarm_qa_tools.md:195]: "autopay and operator configs"

**Review by lpgeiger — APPROVED** [review]
Looks good but just need a couple of spelling checks. Thanks!

**Comment by mortonbits** [comment]
typos fixed :)

**Review by lpgeiger — APPROVED** [review]

**Pull request merged**